.github/workflows/build.yml

```diff
@@ -9,74 +9,30 @@ on:
 permissions:
   contents: read
 
```

Comment:
Sorry spotted one more thing - could you update the `name:` at the top of the workflow from `heroku/hatchet/build` to `CI` or similar? The name the conversion tool picks is quite verbose, and makes the GitHub status checks UI a bit cluttered.